src/slack

```diff
@@ -68,9 +68,9 @@ while (( "$#" )); do
     --filter=*) filter=${1/--filter=/''} ; shift ;;
     --filter*|-f*) filter=${2} ; shift ; shift ;;
     --footer-icon=*) footericon=${1/--footer-icon=/''} ; shift ;;
-    --footer-icon*|-fi*) footericon=${2} ; shift ; shift ;;
+    --footer-icon*) footericon=${2} ; shift ; shift ;;
     --footer=*) footer=${1/--footer=/''} ; shift ;;
-    --footer*|-ft*) footer=${2} ; shift ; shift ;;
+    --footer*) footer=${2} ; shift ; shift ;;
```

Comment:
Here I removed the short form (masked by `-f*` of `--filter`) as I thought these two options would be used less often